clients/js/test/add.collections.test.ts

```diff
@@ -24,21 +24,24 @@ describe("add collections", () => {
 
   beforeEach(async () => {
     await client.reset();
+    // the sleep assures the db is fully reset
```

Comment:
Do we have an issue for this if not I can make one

Reply:
that'd be great, thanks

Reply:
https://github.com/chroma-core/chroma/issues/2774